index.html

```diff
@@ -0,0 +1,193 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png" />
+    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>            
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" type="text/css">
+    <link rel="stylesheet" href="style.css" />
+    
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
+        rel="stylesheet"
+      />
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous"
+        referrerpolicy="no-referrer"
+      />
+      <script
+        async
+        src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"
+      ></script>
+      
+    <title>Jugoslav Capstone Project 1</title>
+</head>
+<body>
+<!-- -------------------------Header section Starts------------------------------>
+<nav class="header-one">
+    <ul class="padding">
+        <li><i class="fab fa-facebook-f"></i></li>
+        <li><i class="fab fa-twitter"></i></li>
+        <li>english</li>
+        <li>my page</li>
+        <li>logout</li>
+    </ul>
+</nav>
+
+   <!--------Header two---------------------------->
+   <header class="header-two">
```

Comment:
- [ ] [ OPTIONAL] I would like to recommend you to improve your mobile version and make sure to stick the original design as much as possible in order to give a good user experience :smile:

Reply:
@codecaiine Thanks for the suggestion. The video is shortened. This the same link
https://www.loom.com/share/69349eb9e8de4197988d23b173c3f82b